wsclient/src/main/java/org/hpccsystems/ws/client/platform/test/BaseRemoteTest.java

```diff
@@ -42,7 +42,7 @@ HPCC SYSTEMS software Copyright (C) 2019 HPCC Systems®.
 
     protected final static String hpccUser = System.getProperty("hpccuser", defaultUserName);
     protected final static String hpccPass = System.getProperty("hpccpass", "");
-    protected final static String connTO = System.getProperty("connecttimeoutmillis");
+    protected final static Integer connTO = System.getProperty("connecttimeoutmillis")==null?null:Integer.valueOf(System.getProperty("connecttimeoutmillis"));
```

Comment:
are all references to connTO in the project null safe?
is the ternary expression needed because Integer.valueOf(null) != null?

Reply:
Integer.valueOf(null) throws an error. Yes, the code handles connTO being null. I found one place it wasn't and fixed it and pushed the change to this PR.